### DISCUSSION TITLE
Which one to which: b0 and T1

### DISCUSSION THREAD

**Discussion by amirhusseinab** [PennLINC/qsiprep#719] [Q&A]
Hi all,

Ever since the beginning of my journey in working with dMRI data, there has been a constant debate regarding the alignment of T1-weighted images with dMRI data, especially when anatomical information is required (e.g., ACT).

My instinct has always leaned towards transforming T1 images into the dMRI space to limit interpolation and decrease potential errors that could emerge during the alignment of FODs in voxels, following the registration of dMRI to T1 images. As far as I know, the QSIprep pipeline adopts a b0-to-T1 registration approach, and I'm eager to hear your insights on the following points:

1) What are the reasons for not adopting the alternative approach of T1-to-b0 registration?
2) Is there a feasible method to implement this using QSIprep?

Best,
Amir

**Comment by mattcieslak** [comment]
Unfortunately 2 would probably be a "no", it would require a lot of reworking the workflows. But I don't think it's that unfortunate because there are good reasons to use the anatomical scan to define the subject-specific anatomical space.

First, the dmri data will always get interpolated. The number of interpolations can be minimized (and that is something qsiprep is very careful about), but if you're going to distortion/eddy/head motion correct, you're going to be interpolating your dmri data. The user picks the voxel size of the output space, so they control the amount of upsampling.

Second, dmri volumes are tricky to use to define subject space with. If you have an AP and PA scan, which should be used to define subject space? Should a b=0 from that scan be used to define the space? Or some summary image of the entire scan? If you pick the AP, you will then need to do an alignment and interpolation of the PA into its FOV. What if the dmri has anisotropic voxels that would need to be resampled for tractography?

Third, possibly the main innovation of qsiprep is that it makes sure that data is accurately shared across many different software packages. One of the most important aspects of this is making sure that none of the data is written out to an oblique voxel grid. If you pick one of the dmri acquisitions, there is a small but real chance that you'll end up with oblique outputs, which behave very differently in the different software packages available in recon mode.

Fourth, qsiprep doesn't resample to the native T1 or T2, but to the AC-PC aligned version of that scan. This is a choice also taken by the HCP pipelines and one that I think makes a lot of sense. Having AC-PC aligned outputs makes visualization and QC much easier. We use the dmriprep viewer or swipes for science for QCing results, and having similarly oriented processed images makes life much easier for raters. AC-PC alignment is also guaranteed to be non-oblique. When using a recon method with angular basis sets (eg spherical harmonics, MAPMRI) you will get similar looking coefficients across subjects, which can help with QC on recon workflows.


I should probably add this to the documentation because we've had a few questions about it.

**Reply by amirhusseinab** [reply]
Hi Matt,

Thank you very much for your elaborate explanation—I also apologize for my late reply.

I get the third and fourth points, but my concern is that the dwi-to-t1 registration may result in disruption of the diffusion orientation in each voxel, and subsequently affecting tractography discussed in forums/papers such as https://analyticalsciencejournals.onlinelibrary.wiley.com/doi/full/10.1002/nbm.1543, and https://www.sciencedirect.com/science/article/pii/S1053811903004981?pes=vor.

What do you think about that?
